tests/integration_tests/test_custom_components.py

```diff
@@ -61,7 +55,15 @@ class CustomLossConfig(BaseLossConfig):
     type: str = "custom_loss"
 
 
-@register_combiner(name="custom_test")
+@register_combiner_schema("custom_combiner")
```

Comment:
Is this going to end up in the combiner registry? (No, right? - since it's under `tests`)

Reply:
That's right.